playbooks/generate-csr.yaml

```diff
@@ -0,0 +1,34 @@
+- hosts: nerc-bootstrap
+  gather_facts: false
+  vars:
+    common_names:
+      - api.nerc-ocp-prod.rc.fas.harvard.edu
+      - apps.nerc-ocp-prod.rc.fas.harvard.edu
```

Comment:
Shouldn't this be `*.apps.nerc-ocp-prod.rc.fas.harvard.edu`?

Reply:
This should be configured for whichever CN you are trying to generate, but you are correct, *.apps should be in there and I have corrected that.